electron/renderer/index.js

```diff
@@ -49,22 +49,24 @@ render(CompassProfilerVisualizePlugin);
 import Connection from 'mongodb-connection-model';
 import DataService from 'mongodb-data-service';
 
+const DB = 'db1';
+
 const connection = new Connection({
   hostname: '127.0.0.1',
-  port: 27000,
```

Comment:
My local instance is on 27000. Could we keep it as is?

Reply:
Sure, just document it so that people running your plugin won't be confused.